spec/features/consumer/shopping/cart_spec.rb

```diff
@@ -143,5 +143,26 @@
         expect(page).to have_content item2.variant.name
       end
     end
+
+    context "Admin & Handling" do
+      it "hides row if order includes no adjustments" do
+        add_product_to_cart order, product_fee, quantity: 1
+        visit spree.cart_path
+
+        expect(page).to have_selector('#cart-detail')
+        expect(page).to_not have_content('Admin & Handling')
+      end
+
+      it "shows row if order includes adjustments" do
+        coordinator_fee = create(:enterprise_fee, enterprise: order_cycle.coordinator, fee_type: 'admin', calculator: Spree::Calculator::FlatRate.new(preferred_amount: 1))
+        order_cycle.coordinator_fees << coordinator_fee
+
+        add_product_to_cart order, product_fee, quantity: 1
+        visit spree.cart_path
+
+        expect(page).to have_selector('#cart-detail')
+        expect(page).to have_content('Admin & Handling')
+      end
+    end
```

Comment:
You can use `context` and `before` blocks to reuse some code and make the specs more readable: 
```ruby
before do
  add_product_to_cart order, product_fee, quantity: 1
end

context 'when there are no fees' do
  it 'hides admin and handlings row' do
    visit spree.cart_path

    expect(page).to have_selector('#cart-detail')
    expect(page).to_not have_content('Admin & Handling')
  end
end

context 'when there are fees' do
  before do
    coordinator_fee = create(:enterprise_fee, enterprise: order_cycle.coordinator, fee_type: 'admin', calculator: Spree::Calculator::FlatRate.new(preferred_amount: 1))
    order_cycle.coordinator_fees << coordinator_fee
  end

  it 'shows admin and handlings row' do
    visit spree.cart_path

    expect(page).to have_selector('#cart-detail')
    expect(page).to have_content('Admin & Handling')
    # Maybe we can also check the amount since we're here
  end
end